docs/proposals/20181121-machine-api.md

```diff
@@ -41,7 +41,7 @@ performed in-place or via Node replacement.
 ## Proposal
 
 This proposal introduces a new API type: Machine. See the full definition in
-[machine_types.go](../../api/v1alpha4/machine_types.go).
+[machine_types.go](https://github.com/kubernetes-sigs/cluster-api/blob/release-0.4/api/v1alpha4/machine_types.go).
```

Comment:
Makes sense to me to link to the same generation of types the proposal was talking about.